python/tests/test_mg_gaas_handler.py

```diff
@@ -0,0 +1,276 @@
+
+
+def test_get_graph_info(handler_with_karate_edgelist_loaded):
+    """
+    get_graph_info() for specific args.
+    """
+    from gaas_client import defaults
+    from gaas_client.types import ValueWrapper
+
+    (handler, test_data) = handler_with_karate_edgelist_loaded
+
+    # A common use of get_graph_info() is to get the "shape" of the data,
+    # meaning the number of vertices/edges by the number of properites per
+    # edge/vertex.
+    info = handler.get_graph_info(["num_edges", "num_edge_properties"],
+                                  defaults.graph_id)
+    # info is a dictionary containing gaas_client.types.Value objs, so access
+    # the int32 member directly for easy comparison.
+    shape = (ValueWrapper(info["num_edges"]).get_py_obj(),
+             ValueWrapper(info["num_edge_properties"]).get_py_obj())
+    assert shape == (156, 1)  # The single edge property is the weight
+
+    info = handler.get_graph_info(["num_vertices_from_vertex_data",
+                                   "num_vertex_properties"],
+                                  defaults.graph_id)
+    shape = (ValueWrapper(info["num_vertices_from_vertex_data"]).get_py_obj(),
+             ValueWrapper(info["num_vertex_properties"]).get_py_obj())
+    assert shape == (0, 0)
+
+
+def test_get_graph_info_defaults(mg_handler):
+    """
+    Ensure calling get_graph_info() with no args returns the info dict with all
+    keys present for an empty default graph.
+    """
+    from gaas_client import defaults
+    from gaas_client.types import ValueWrapper
+
+    handler = mg_handler
+
+    info = handler.get_graph_info([], graph_id=defaults.graph_id)
+
+    expected = {"num_vertices": 0,
+                "num_vertices_from_vertex_data": 0,
+                "num_edges": 0,
+                "num_vertex_properties": 0,
+                "num_edge_properties": 0,
+                }
+    actual = {key:ValueWrapper(val).get_py_obj() for (key, val) in info.items()}
+
+    assert expected == actual
+
+
+def test_uniform_neighbor_sampling(handler_with_karate_edgelist_loaded):
+    from gaas_client.exceptions import GaasError
+    from gaas_client import defaults
+
+    (handler, test_data) = handler_with_karate_edgelist_loaded
+
+    start_list = [1, 2, 3]
+    fanout_vals = [2, 2, 2]
+    with_replacement = True
+
+    # invalid graph type - default graph is a PG, needs an extracted subgraph
+    with pytest.raises(GaasError):
+        handler.uniform_neighbor_sample(start_list=start_list,
+                                        fanout_vals=fanout_vals,
+                                        with_replacement=with_replacement,
+                                        graph_id=defaults.graph_id)
+
+    extracted_gid = handler.extract_subgraph(create_using=None,
+                                             selection=None,
+                                             edge_weight_property=None,
+                                             default_edge_weight=1.0,
```

Comment:
This will probably be `edge_id_n` for DGL workflows in the future, I hope we can handle that too ?  

https://github.com/rapidsai/cugraph/blob/821571d6c2635506c81c4da92d73a785f3798862/python/cugraph/cugraph/gnn/graph_store.py#L282-L286